build failed at latest zig release

log
```
install
└─ install arocc
   └─ zig build-exe arocc ReleaseFast native 3 errors
src/aro/target.zig:452:5: error: switch must handle all possibilities
    switch (target.cpu.arch) {
    ^~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:1165:9: note: unhandled enumeration value: 'propeller1'
        propeller1,
        ^~~~~~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:1166:9: note: unhandled enumeration value: 'propeller2'
        propeller2,
        ^~~~~~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:1135:22: note: enum 'Target.Cpu.Arch' declared here
    pub const Arch = enum {
                     ^~~~
src/aro/target.zig:508:5: error: switch must handle all possibilities
    switch (target.cpu.arch) {
    ^~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:1165:9: note: unhandled enumeration value: 'propeller1'
        propeller1,
        ^~~~~~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:1166:9: note: unhandled enumeration value: 'propeller2'
        propeller2,
        ^~~~~~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:1135:22: note: enum 'Target.Cpu.Arch' declared here
    pub const Arch = enum {
                     ^~~~
referenced by:
    discover: src/aro/Driver/GCCDetector.zig:405:40
    discover: src/aro/toolchains/Linux.zig:19:35
    7 reference(s) hidden; use '-freference-trace=9' to see all references
src/aro/target.zig:570:23: error: switch must handle all possibilities
    const llvm_arch = switch (target.cpu.arch) {
                      ^~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:1165:9: note: unhandled enumeration value: 'propeller1'
        propeller1,
        ^~~~~~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:1166:9: note: unhandled enumeration value: 'propeller2'
        propeller2,
        ^~~~~~~~~~
/usr/lib/zig/9999/lib/std/Target.zig:1135:22: note: enum 'Target.Cpu.Arch' declared here
    pub const Arch = enum {
                     ^~~~
error: the following command failed with 3 compilation errors:
/usr/lib/zig/9999/bin/zig build-exe -fsingle-threaded -OReleaseFast --dep aro -Mroot=/home/misile/arocc/src/main.zig --dep system_defaults --dep build_options --dep backend --dep Builtins/Builtin.def --dep Attribute/names.def --dep Diagnostics/messages.def -Maro=/home/misile/arocc/src/aro.zig -Msystem_defaults=/home/misile/arocc/.zig-cache/c/f3b600c007f1c4b76559c1bd481127ba/options.zig -Mbuild_options=/home/misile/arocc/.zig-cache/c/2f140dafa880198dcfbad23a03cd0305/options.zig --dep zig --dep build_options -Mbackend=/home/misile/arocc/src/backend.zig -MBuiltins/Builtin.def=/home/misile/arocc/.zig-cache/o/71868f40b58f45c3bfdc7f62fd365cc9/Builtin.zig -MAttribute/names.def=/home/misile/arocc/.zig-cache/o/bfe5f5e8adcdb7548f69eafae84349e3/names.zig -MDiagnostics/messages.def=/home/misile/arocc/.zig-cache/o/77570701ca6862771c16e46a833c174e/messages.zig -Mzig=/home/misile/arocc/deps/zig/lib.zig -lc --cache-dir /home/misile/arocc/.zig-cache --global-cache-dir /home/misile/.cache/zig --name arocc --zig-lib-dir /usr/lib/zig/9999/lib/ --listen=-
Build Summary: 6/9 steps succeeded; 1 failed
install transitive failure
└─ install arocc transitive failure
   └─ zig build-exe arocc ReleaseFast native 3 errors
error: the following build command failed with exit code 1:
/home/misile/arocc/.zig-cache/o/8936f1a517aa90946826a1884d2336cd/build /usr/lib/zig/9999/bin/zig /usr/lib/zig/9999/lib /home/misile/arocc /home/misile/arocc/.zig-cache /home/misile/.cache/zig --seed 0x5097fc88 -Zcd4e4920f006ca6b -Doptimize=ReleaseFast
```
zig version: `0.14.0-dev.1767+d23db9427`

--- a/src/aro/target.zig
+++ b/src/aro/target.zig
@@ -653,7 +653,6 @@ pub fn toLLVMTriple(target: std.Target, buf: []u8) []const u8 {
         .tvos => "tvos",
         .watchos => "watchos",
         .driverkit => "driverkit",
-        .shadermodel => "shadermodel",
         .visionos => "xros",
         .serenity => "serenity",
         .opencl,

